_articles/managing-metadata.md

```diff
@@ -0,0 +1,53 @@
+---
+title: Managing Metadata
+layout: article
+excerpt: Merge individual, biospecimen and assay-specific metadata files.
+category: tutorials
+---
+
+File annotations and metadata provide critical information about the data in the AD Knowledge portal. Annotations describe content within the files, including: type of data and assay, tissue of origin, source study, ID of the source individual and type of data. Annotations conform to a structured schema to make data highly searchable and standardized across studies and projects. The standards for defining, managing and implementing controlled vocabularies for content in Synapse is available in the synapseAnnotations Github repository. 
```

Comment:
> Annotations conform to a structured schema to make data highly searchable and standardized across studies and projects.

Both metadata and annotations conform to the structured schema. Not sure where to put that in this paragraph, but both are mentioned in the first sentence.

> synapseAnnotations Github repository

Could use a link here.

Reply:
metadata doesn't always conform to the structured schema. I will add the link. thanks!